docs/serving/samples/knative-routing-go/README.md

```diff
@@ -174,8 +189,26 @@ You should see: `Login Service is called !`
 kubectl apply --filename docs/serving/samples/knative-routing-go/routing.yaml
 ```
 
+f you have configured a custom domain name for your service, please replace all
```

Comment:
Missing an "I" at the beginning on this sentence.

Reply:
Done, thanks.